src/libraries/apicompat/ApiCompatBaseline.NetCoreAppLatestStable.xml

```diff
@@ -2641,4 +2641,10 @@
     <Left>net7.0/System.Runtime.Loader.dll</Left>
     <Right>net8.0/System.Runtime.Loader.dll</Right>
   </Suppression>
+  <Suppression>
```

Comment:
Do we need both suppression file changes?

Reply:
It was put there by the tool, so I figure we probably need both.

Reply:
Yes. This one is for apicompat.proj which compares the netstandard2.x and net7.0 reference assemblies with net8.0 and lists all breaking changes.

The other suppression file is directly in System.Text.Json for the package and ref <--> src comparison. I don't think that we need to share the suppression, especially given that this (NetCoreAppLatestStable.xml) suppression file will be reset when we branch off to net9.0 (as by that time we will have zero breaking changes between net8.0 and net9.0).